operatorconfig/driverconfig/common/default.yaml

```diff
@@ -24,4 +24,7 @@ images:
   externalhealthmonitorcontroller: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.8.0
   
   # "images.sdcmonitor" defines the container images used to monitor sdc container
-  sdcmonitor: dellemc/sdc:3.6.0.6
\ No newline at end of file
+  sdcmonitor: dellemc/sdc:3.6.0.6
+
+  #"images.metadataretriever" defines the container images used for csi metadata retriever
```

Comment:
Please add this sidecar in all K8s samples supported by 2.7.0 CSI driver

Reply:
updated the files .